backend/localstack/initialize.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Script was failing without the shebang. See https://stackoverflow.com/a/27608363.